mailpoet/lib/Analytics/UnsubscribeReporter.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace MailPoet\Analytics;
+
+use MailPoet\Entities\StatisticsUnsubscribeEntity;
+use MailPoet\Statistics\StatisticsUnsubscribesRepository;
+
+class UnsubscribeReporter {
+
+  public const TOTAL = 'Unsubscribe > Total in last 6 months';
+  public const COUNT_PER_METHOD_PATTERN = 'Unsubscribe > Count in last 6 months with method: %s';
+
+  /*** @var StatisticsUnsubscribesRepository */
+  private $statisticsUnsubscribesRepository;
+
+  public function __construct(
+    StatisticsUnsubscribesRepository $statisticsUnsubscribesRepository
+  ) {
+    $this->statisticsUnsubscribesRepository = $statisticsUnsubscribesRepository;
+  }
+
+  public function getProperties(): array {
+    $properties = [
+      self::TOTAL => $this->statisticsUnsubscribesRepository->getTotalForMonths(6),
+    ];
+
+    foreach ($this->statisticsUnsubscribesRepository->getCountPerMethodForMonths(6) as $methodStats) {
+      $properties[sprintf(self::COUNT_PER_METHOD_PATTERN, $this->getMethodName($methodStats['method']))] = $methodStats['count'];
+    }
+
+    return $properties;
+  }
+
+  /**
+   * @param string|null $methodKey
+   */
+  private function getMethodName($methodKey): string {
+    if ($methodKey === StatisticsUnsubscribeEntity::METHOD_ONE_CLICK) {
+      return '1 Click';
+    }
+
+    if ($methodKey === StatisticsUnsubscribeEntity::METHOD_LINK) {
+      return 'Link';
+    }
+
+    return 'Unknown';
```

Comment:
Just an idea. We already [track admin names when tracking unsubscribers](https://github.com/mailpoet/mailpoet/blob/76048224d112a46c1a1d60873694798e351600f5/mailpoet/lib/Subscribers/SubscriberSaveController.php#L139) so maybe we could distingush some value in meta as method `admin` and `null` as `Unknown`.